src/handlers/album_tracks.rs

```diff
@@ -53,6 +51,60 @@ pub fn handler(key: Key, app: &mut App) {
                 }
             }
         },
+        k if common_key_events::high_event(k) => match app.album_table_context {
```

Comment:
To fix the clippy complexity, you could move the handler logic into separate functions e.g.

```rust
fn handle_high_event(app: &mut App) {
  match app.album_table_context {
           AlbumTableContext::Full => {
                if let Some(albums) = &app.library.clone().saved_albums.get_results(None) {
                    if let Some(_selected_album) = albums.items.get(app.album_list_index) {
                        let next_index = common_key_events::on_high_press_handler();
                        app.saved_album_tracks_index = next_index;
                    }
                };
            }
            AlbumTableContext::Simplified => {
                if let Some(selected_album) = &mut app.selected_album {
                    let next_index = common_key_events::on_high_press_handler();
                    selected_album.selected_index = next_index;
                }
            }
        },
}

...
k if common_key_events::high_event(k) => handle_high_event(app),
```

Reply:
Sounds good. I should be able to wrap this up tonight.